src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -33,6 +33,8 @@ const WithContexts = Component => {
       isNextJs,
     } = props;
 
+    const analytics = path(['metadata', 'analytics'], pageData);
+
```

Comment:
Can we use object destructing instead of `path` (we're unofficially trying to reduce usage of ramda)?
Can we fetch the title from the pageData here, and also construct the atiData object?

```suggestion
    const { title, metadata: analytics } = pageData;
    
    let atiData;
    if (analytics) {
      atiData = { analytics, title };
    }
```

Reply:
Done and utilising optional chaining in place of ramda (extended it for all instances of ramda in the file)